src/tests/test_system.py

```diff
@@ -181,7 +181,7 @@ def test_thermo_bromine_study(self):
 
 class TestEvaluatorBromineStudy(ForceBalanceSystemTest):
     def setup_method(self, method):
-        pytest.importorskip("evaluator")
+        pytest.importorskip("openff.evaluator")
```

Comment:
(not blocking) This is confusing. The previous import test should have always failed, so this test should have always skipped, but I was seeing this test failure on my computer before you made this change. So this may be "smoke" for something deeper, but it's not blocking because 1) my memory is bad and I could be mistaken and 2) we fixed the study by running it explicitly so we know that it works.

Reply:
That's definitely strange. It should be fixed now, and I don't see why the test would have failed unless your env had the old package name.